codersdk/toolsdk/bash_test.go

```diff
@@ -1,3 +1,6 @@
+//go:build !windows
```

Comment:
Could we instead skip each test in this file with an explanation? When reading it's easy to miss build exclusions. I'm sure somebody in the future will wonder why our workspace MCP tools don't work on Windows.